services/contract_update.proto

```diff
@@ -95,5 +100,16 @@ message ContractUpdateTransactionBody {
        */
       google.protobuf.StringValue memoWrapper = 10;
     }
-}
 
+    /**
+     * If set, the new maximum number of tokens that this contract can be 
+     * automatically associated with (i.e., receive air-drops from).
+     */
+    google.protobuf.Int32Value max_automatic_token_associations = 11;
+
+    /**
+     * If set, either gives the value of the new auto-renew account id for this contract; or if 
+     * set without any inner value, removes the existing auto-renew account of the contract.
+     */
+    AccountIDValue auto_renew_account_id = 12;
```

Comment:
Just spitballing ideas, what if instead we added a `bool clear` inside `AccountID`? Is that weird?